src/main/java/org/mastodon/mamut/feature/branch/BranchNSpotsFeature.java

```diff
@@ -46,13 +41,141 @@
 import org.mastodon.properties.IntPropertyMap;
 import org.scijava.plugin.Plugin;
 
+import java.util.Collections;
+import java.util.Set;
+
+import static org.mastodon.feature.FeatureProjectionKey.key;
```

Comment:
Please avoid resorting the import statements.

Reply:
Changed it. And put on the fly optimizing imports of.